ddp-workspace/projects/ddp-dsm-ui/src/app/services/auth.service.ts

```diff
@@ -227,4 +229,13 @@ export class Auth {
       this.router.navigate([ Statics.HOME_URL ]);
     }
   }
+
+  private setDssSession(dsmToken: string): void {
+    const accessToken = null;
+    const userGuid = null;
+    const locale = 'en';
+    const expiresAtInSeconds = 1e10; // TODO: get rid of that by using a study config flag as to not renew a session
```

Comment:
Well, we should extract from the token itself the expiration, no?

Reply:
I thought a DSS token should never expire for DSM operator/admin
(means that a DSM "admin" can use DSS without time limit). 
Am I wrong?

Reply:
The admin has more privileges to do stuff, but their tokens also expire.
I am under distinct impression that DSM token includes an expiration.
(Which brings up the question of what we should do if the token really has expired. We probably want to do whatever DSM currently does).

Reply:
Agree, 
fixed in db71d57852287276b3fd264612ed1b2d91eaccb4